src/components/Fuel/FuelForm.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react';
+import { Button, Label, Select, TextInput } from 'flowbite-react';
+import {
+  FuelSourceUnit,
+  FuelSourceType,
+  UnregisteredFuelRequest,
+} from '../../schema/fuel.schema';
+
+interface FuelFormProps {
+  handleSubmit: (fuelData: UnregisteredFuelRequest) => void;
+}
+
+export default function FuelForm({ handleSubmit }: FuelFormProps) {
+  /* Initializing state */
+  const [fuelData, setfuelData] = useState({
+    fuel_source_type: 'ng' as FuelSourceType,
+    fuel_source_unit: 'btu' as FuelSourceUnit,
+    fuel_source_value: 0 as number,
+  });
+
+  function submitForm(e: React.FormEvent) {
```

Comment:
Be consistent with your variable names. You have `e` here and `event` at other places.